### PR TITLE
Fixed creating a new organization

### LINKED ISSUE #1139: Cant add new organisation
### Subject of the issue
When I go to add new organisation it just continually loads in the logs I get:
[2020-09-13 09:03:33.207][response][INFO] GET /api/sync?<data..> (sync) => 200 OK
[2020-09-13 09:03:34.952][request][INFO] GET /api/plans/
[2020-09-13 09:03:34.952][response][INFO] GET /<p..> [10] (web_files) => 404 Not Found

### Your environment
Bitwarden_rs version: 1.16.3
* Install method: built from source using install script
* Clients used: chrome on mac
* Reverse proxy and version: nginx/1.18.0 (Ubuntu)
* Other relevant information: 

### Steps to reproduce
You can run the install script on any machine running ubuntu 20.04 https://github.com/dinger1986/bitwardenrs_install_script/blob/master/bitwardenrs.sh

### Expected behaviour
Should be able to add organisation but cant

### Actual behaviour
Web page sits spinning

### Relevant logs
[2020-09-13 09:19:47.564][request][INFO] GET /api/accounts/revision-date
[2020-09-13 09:19:47.565][response][INFO] GET /api/accounts/revision-date (revision_date) => 200 OK
[2020-09-13 09:19:49.843][request][INFO] GET /api/accounts/profile
[2020-09-13 09:19:49.844][response][INFO] GET /api/accounts/profile (profile) => 200 OK
[2020-09-13 09:19:52.511][request][INFO] GET /api/sync?excludeDomains=true
[2020-09-13 09:19:52.512][response][INFO] GET /api/sync?<data..> (sync) => 200 OK
[2020-09-13 09:19:53.657][request][INFO] GET /api/plans/
[2020-09-13 09:19:53.657][response][INFO] GET /<p..> [10] (web_files) => 404 Not Found
[2020-09-13 09:19:57.579][ws::handler][DEBUG] Handler received: 
<FRAME>
final: true
reserved: false false false
opcode: BINARY
length: 5
payload length: 3
payload: 0x2916
Screenshot 2020-09-13 at 10 19 31

### PR DESCRIPTION
**Pull request by BlackDex**
- The new web-vault needs a new api endpoint.
- Added this new endpoint.

Fixes #1139

### CODE REVIEW

File: src/api/core/organizations.rs [hunk 1]
```diff
@@ -991,3 +992,57 @@ fn put_policy(org_id: String, pol_type: i32, data: Json<PolicyData>, _headers: A
 
     Ok(Json(policy.to_json()))
 }
+
+#[get("/plans")]
+fn get_plans(_headers: Headers, _conn: DbConn) -> JsonResult {
+    let plans = json!({
```
Comment (jjlin) [inline src/api/core/organizations.rs:998]:
```suggestion
    Ok(Json(json!({
```

**Comment by jjlin** [comment]
Will just returning the free plan break any features? It looks like there are some features gated by the plan info, e.g.

https://github.com/bitwarden/web/blob/ce67497d3a4faccc6c816959b4a0884917e68a4f/src/app/organizations/tools/tools.component.ts#L25

https://github.com/bitwarden/web/blob/ce67497d3a4faccc6c816959b4a0884917e68a4f/src/app/tools/cipher-report.component.ts#L79

and maybe others.

**Comment by BlackDex** [comment]
I will change the return part, that is a good one indeed.
It doesn't matter what we use here, since we do not use that info anyway, and return our own org usages/details back.

**Comment by BlackDex** [comment]
Also, the plan info is used to display different kind of plans when creating an organization.
I think bitwarden.com is making this more flexible.
They also changed the plans, disabled old once and created new once.
The Plan ID we currently use for example has been disabled:
See the attachment.

[full_plans.json.txt](https://github.com/dani-garcia/bitwarden_rs/files/5216524/full_plans.json.txt)

**Comment by jjlin** [comment]
Okay, looks like just returning the free plan should be fine for the `/plans` response, but in the future, the org response model might need to include the plan details:

* https://github.com/bitwarden/server/blob/18808893250c1236d109c25b5f6a9ec9b33ce70e/src/Core/Models/Api/Response/OrganizationResponseModel.cs#L30
* https://github.com/bitwarden/server/blob/master/src/Core/Models/Api/Response/PlanResponseModel.cs

**Comment by BlackDex** [comment]
We have to fully check what impact this will have on the web-vault etc..
But yea, we should probably add all the "missing" items and even update the plan type from 5 to something else to more closely match the features we have enabled to prevent other issues if they are going to check those.

**Pull request merged**